tfe/resource_tfe_team_members.go

```diff
@@ -14,6 +14,9 @@ func resourceTFETeamMembers() *schema.Resource {
 		Read:   resourceTFETeamMembersRead,
 		Update: resourceTFETeamMembersUpdate,
 		Delete: resourceTFETeamMembersDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
```

Comment:
This would be nice to use! Import doesn't work at all here. `resourceTFETeamMembersRead` doesn't use the id, just the `team_id`.

ex: `tf import tfe_team_members.test "team-2WbNREnViCFK17hP"`

```
* import tfe_team_members.test result: team-2WbNREnViCFK17hP: tfe_team_members.test: Error reading users from team : Invalid value for team ID
```

Reply:
Fixed in the latest version (will push the commit when I'm done).